app/routers/api/identity.py

```
@@ -0,0 +1,112 @@
+)
+from app.security import PermissionChecker, verify_password
+from app.services.identity_service import identity_service
+
+logger = logging.getLogger(__name__)
+
+router = APIRouter(prefix="/api/v1/identity", tags=["Identity"])
+
+
+@router.get("/public-export", response_model=IdentityPublicExportResponse)
+def export_public_keys(
+    db: Session = Depends(get_db),
```

Ruff (0.14.14)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)